scripts/apps/search/directives/ItemList.js

```diff
@@ -399,6 +399,14 @@ export function ItemList(
                 }
 
                 elem.on('keydown', listComponent.handleKey);
+
+                // detects if its a manual scroll
+                elem.on('scroll mousedown wheel keyup', (e) => {
```

Comment:
I don't think you need this `userHasScrolled` flag. Here, you have two handlers for the scroll event. This one (that I'm commenting on), and the `handleScroll` function below.

If you want to listen to these extra events, just add them below and introduce the extra logic into `handleScroll`. Try to get read of the boolean flag. There are many other ways to figure out if the user has scrolled.